src/main/java/at/hannibal2/skyhanni/utils/InventoryUtils.kt

```diff
@@ -131,6 +134,18 @@ object InventoryUtils {
         }
     }
 
+    fun Gui.getTitle() = when (this) {
```

Comment:
please avoid using `!!`. rather use ` ?: error("bla bla")` or use the magic file somewehre to mark those fields as non null globally